### DISCUSSION TITLE
Newer cards not appearing in favourite or favoured lists for some tracks

### DISCUSSION THREAD

**Discussion by dtra82** [MKT-Playbook/Hub#18] [General]
![16B7FCC3-4120-4038-8059-55B07DC130CC](https://user-images.githubusercontent.com/116084662/205772913-f4c9a547-7889-47dd-acee-ecde3c0c1ba5.png)
The moo moo offroader has a level 8 unlock on this track, doesn’t appear on either list, I don’t own it but it should still appear. Seems to be the case for new-ish items in general, not just this kart.

**Comment by TitusPosiadly** [comment]
I just checked and it is appearing in my list. This is likely caused by issue #15. I will refresh the database so your app can get the newest data. Let me know if you still have trouble after that (should only take a couple of minutes)

**Reply by dtra82** [reply]
Still looks the same for me

**Reply by TitusPosiadly** [reply]
Hmm, strange. Can you take a look if it's the same the other way around? So does Riverside park appear in the list of tracks for the moo moo offroader?

**Reply by dtra82** [reply]
yeah, all the tracks appear correctly for the item(s). Version is 1.21 (56) btw.

**Reply by TitusPosiadly** [reply]
Can you tell me if the main device in the settings is the device you are currently using? (It should say "This Device" in brackets after the device name)

**Reply by dtra82** [reply]
Yep, it is the main device.

**Reply by TitusPosiadly** [reply]
When you go to the shops or pipes do you see a prompt telling you "Scan missing cards to see their coverage"? Sorry for all the questions btw. I'm really having trouble figuring this out as I cannot seem to replicate the issue. I appreciate the help!

**Reply by dtra82** [reply]
Ok, that did it, thanks.
